Clarify OTA terms in README.md (#6)
OpenTermsArchive/demo-versions: README.md

# License

> Any database in this repository is distributed under an ODC-BY 1.0 license. That means you are free to share, to create, to adapt the database as long as you attribute the resulting works to *<!-- customize to your context -->Open Terms Archive Core Team<!-- until here --> and Open Terms Archive contributors*.

Reviewer: ```suggestion
Any database in this repository is distributed under an [ODC-BY 1.0](https://opendatacommons.org/licenses/by/1-0/) license. That means you are free to share, to create, to adapt the database as long as you attribute the resulting works to *<!-- customize to your context -->Open Terms Archive Core Team<!-- until here --> and Open Terms Archive contributors*.
```

> Any database in this repository is distributed under an ODC-BY 1.0 license. That means you are free to share, to create, to adapt the database as long as you attribute the resulting works to *<!-- customize to your context -->Open Terms Archive Core Team<!-- until here --> and Open Terms Archive contributors*.

Reviewer: When removing the details in parentheses, I find “adapt” and even more so “create” much less clear. Could we bring back the parentheses, or maybe at least write “create from”? 🙂

Author: How about "That means you are free to use, to share, to modify or adapt, and to create or produce works from the database as long as ..." ? 

I think the parentheses make it harder to follow, but see that the previous version cut out too much.

> Any database in this repository is distributed under an [ODC-BY 1.0](https://opendatacommons.org/licenses/by/1-0/) license. That means you are free to share, to create, to adapt the database as long as you attribute the resulting works to *<!-- customize to your context -->Open Terms Archive Core Team<!-- until here --> and Open Terms Archive contributors*.

Reviewer: ```suggestion
Any database in this repository is distributed under an [ODC-BY 1.0](https://opendatacommons.org/licenses/by/1-0/) license. That means you are free to share, modify, transform the database and produce works from it as long as you attribute the resulting works to *<!-- customize to your context -->Open Terms Archive Core Team<!-- until here --> and Open Terms Archive contributors*.
```

Author: Looks good to me - except perhaps needs a small 'or' added in:

"free to share, modify, or transform the database and produce works from it"

thankyou!

Reviewer: Sounds good! Please apply those changes and we can proceed with merging 🚀

Author: Done :) ready to be merged
